src/component/feature/FeatureStrategy/FeatureStrategyForm/FeatureStrategyForm.tsx

```diff
@@ -118,6 +117,18 @@ export const FeatureStrategyForm = ({
                 )}
             />
             <div className={styles.buttons}>
+                <PermissionButton
```

Comment:
Could use `CreateButton` or `UpdateButton` for consistency.

Reply:
With the current structure we can't effectively do that because it will impact both the Create and Edit version of the form. We'll need to restructure the FeatureStrategyForm to be able to use this approach.